File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/impressao_texto.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class ImpressaoTexto extends TermicaCommand {
  final String _dados;
  final int _posicao;
  final int _stilo;
  final int _tamanho;

  ImpressaoTexto(this._dados, this._posicao, this._stilo, this._tamanho)
      : super('ImpressaoTexto');

  @override
  functionParameters() {
    return '"dados"'
        ':'
        '"$_dados"'
        ','
        '"posicao"'
        ':'
        '$_posicao'
        ','
        '"stilo"'
        ':'
        '$_stilo'
        ','
        '"tamanho"'
        ':'
        '$_tamanho';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/Commands/ImpressaoTexto.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImpressaoTexto(
    private val dados: String,
    private val posicao: Int,
    private val stilo: Int,
    private val tamanho: Int
) : TermicaCommand("ImpressaoTexto") {
    override fun functionParameters(): String {
        return "\"dados\"" + ":" + "\"" + dados + "\"" + "," +
                "\"posicao\"" + ":" + posicao + "," +
                "\"stilo\"" + ":" + stilo + "," +
                "\"tamanho\"" + ":" + tamanho
    }
}